Added doc strings for AdminDao class and its functions

sarnava1 opened the pull request:
### Description
Added doc strings for AdminDao class and its functions
This PR was created after there were some errors during the squashing of commits in the previous PR which had tried to do the same work. The link is https://github.com/systers/mentorship-backend/pull/157.

Fixes #153 

### Type of Change:

- Quality Assurance
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

sarnava1 commented:
@isabelcosta I think this PR is fine now. Please review it.

isabelcosta commented:
When you create a new PR like this related to another PR or issue relate them in some way to keep track of the history. Please update pr description with a link to the other PR @sarnava1

sarnava1 commented:
@isabelcosta done.

sarnava1 commented:
Please review @isabelcosta.

isabelcosta commented on app/api/dao/admin.py:
```diff
@@ -2,9 +2,20 @@
 
 class AdminDAO:
-
+    """Data Access Object for Admin functionalities."""
```
Space below this line please.

isabelcosta replied:
one line below only

isabelcosta reviewed (changes requested):
@sarnava1 Left one change request.

Also the commit message does not have a space between "docs" and ":" so:
Before:
```
docs : added doc strings for AdminDao class and its functions
```

After:
```
docs: added doc strings for AdminDao class and its functions
```

If you get any problem with git or terminal stuff, tell me.
I'll be more than happy to help you :)

sarnava1 commented:
@isabelcosta please review the PR. I will squash the commits if you give the green signal.

isabelcosta reviewed (dismissed):
Green signal given!

isabelcosta commented:
@sarnava1 remember about the comment I made on the commit message. Your 2° commit didn't change

sarnava1 commented:
@isabelcosta Actually I could not understand what I should do!! Initially I gave "docs: added ...." and now I gave "docs : added ..." 

Which should I finally give?

isabelcosta commented:
First option you mentioned @sarnava1

sarnava1 commented:
Ok @isabelcosta

sarnava1 commented:
@isabelcosta I am getting this error:

![error_1](https://user-images.githubusercontent.com/25319565/50548333-bee41580-0c70-11e9-8618-fa41a0241f56.png)

isabelcosta commented:
@sarnava1 I'm on my phone now, can you share that on slack please. I'm sure someone will help you out!

sarnava1 commented:
@isabelcosta sure!

isabelcosta commented:
Uh wait. Always check if you can fix Codacy warnings. You have one error pending @sarnava1 fix that please

sarnava1 commented:
@isabelcosta the codacy warning is because of the blank line I inserted as you asked me to do. Initially I did not put that blank line because it was giving the codacy errors.

isabelcosta commented:
@sarnava1 I verified now that the trailing whitespace problem isn't related to the new line but with the space characters you have in that blank line:

![image](https://user-images.githubusercontent.com/11148726/50549144-5df81580-0c4f-11e9-87b2-cf9bbb63c7ac.png)

highlighted in blue is the trailing whitespace. You can add a blank line without spaces in that line. Do you understand @sarnava1?

sarnava1 commented:
Ok @isabelcosta I am trying it out.

sarnava1 commented:
@isabelcosta the checks have passed!!

isabelcosta commented:
Yey! @sarnava1 you learned something here ;)

isabelcosta commented:
Ok so now you can try to squash the commits :)

sarnava1 commented:
@isabelcosta I learnt a lot here!! I will try to squash the commits.

sarnava1 commented:
@isabelcosta I am getting the same error again!! I have posted it in the slack channel too!

sarnava1 committed:
docs: added doc strings for AdminDao class and its functions

isabelcosta reviewed (approved):
Good job @sarnava1 ! Thank you for your patient :) I'm glad you got to solve your git problem!
